lib/features/home_screen/widgets/contact_icon_widget.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:flutter/material.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+import '../../../config/app_ui_config.dart';
+import '../../../theme/app_theme.dart';
+
+class ContactIconWidget extends StatelessWidget {
+  const ContactIconWidget({super.key, required this.url});
+
+  final String url;
+
+  @override
+  Widget build(BuildContext context) {
+    final Uri _url = Uri.parse('https://flutter.dev');
```

Comment:
Here probably should be, passed in constructor ,`url` variable

Reply:
it should either be 
```dart
final url = Uri.parse(url);
```
or if we're using util like pasted above/below this line can be entirely deleted and just use util below with
`LaunchUrlUtil.lauch(url)`

Reply:
also note here that leading underscore (private access) in local variables doesn't change anything (has no effect), hence not much sense of using this

Reply:
And dart has type inference and I'm fan of using inference if it works nicely, so you can skip type here and it won't affect anything (the type will be inferenced )
```dart
final Uri _url = //...
```